webapp/app/__init__.py

```diff
@@ -42,8 +40,12 @@
 login_manager.init_app(app)
 login_manager.session_protection = 'strong'
 
-# Set pool_pre_ping to fix problems with stale connection we've been seeing.
-db = SQLAlchemy(app, engine_options={'pool_pre_ping': True})
+# SQLAlchemy options
+# - pool_pre_ping: fix problems with stale connection we've been seeing (see also:
+#   https://docs.syseleven.de/syseleven-stack/de/reference/network/known-issues#idle-tcp-sessions-being-closed).
+# - hide_parameters: don't log any parameters with errors or when logging SQL statements (
+#   https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine.params.hide_parameters)
```

Comment:
I've had a look into the way we can or should hide parameters with SQLAlchemy and found [this](https://github.com/sqlalchemy/sqlalchemy/issues/4806#issuecomment-699182814):
>From the perspective of HIPAA compliance and reducing leakage of PHI/PII into logs, this feature doesn't seem to be sufficient, since when the database drive raises an error, that error is included in the StatementError message. This means that a bug on inserting a record where a non-nullable column was set to null can still leak all the other values of columns in that row into the logs.

I am not sure if that would be a real problem for us. However, I wanted to raise it to you and I think we should maybe have a quick talk about it as this might affect the privacy. They describe a solution using to strip everything after `...row contains...` using the `erase_parameters` function (s. [here](https://github.com/sqlalchemy/sqlalchemy/issues/4806#issuecomment-521501106))

Reply:
> It's a problem that already exists
Okay, I was not sure if that problem already existed or if it is introduced through logging the Info level.

Reply:
My understanding is that this conversation is about parameters being included in exceptions, which we would log **if** they're unhandled. Is so, this is a problem we've always had.